SPECS/bind/bind.spec

```diff
@@ -32,9 +32,6 @@ Source13:       named.rwtab
 Source14:       named-chroot.files
 Source15:       https://gitlab.isc.org/isc-projects/dlz-modules/-/archive/main/dlz-modules-main.tar.gz
 
-Patch0:         nongit-fix.patch
-Patch1:         fix-maybe-uninitialized-warning-in-dlz_mysqldyn_mod.patch
```

Comment:
@kgodara912, could you add an explanation in the summary why we could remove these patches now? Is the uninitialized warning fixed now upstream?